README.md

```diff
@@ -50,9 +50,9 @@ vars:
     reddit_ads_schema: your_schema_name 
 ```
 
-## (Optional) Step 4: Additional configurations
-<details><summary>Expand for configurations</summary>
-
+## (Optional) Step 4: Additional configurations 
+<details><summary>Expand for configurations</summary> 
+ 
```

Comment:
After adding the passthrough metric functionality to the report models, we will want to document them within the additional configurations section.

See how they are documented [here](https://github.com/fivetran/dbt_amazon_ads_source#passing-through-additional-metrics) for Amazon Ads.

Reply:
Very minor, but can you reorder the variables to be consistent with the hierarchy: account, campaign, ad group, ad.